scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd

```diff
@@ -13,6 +13,7 @@ SPDX-License-Identifier: Apache-2.0
         <xs:documentation xml:lang="en">
             Version 1.0 Release 2021/06/16
             Version 1.1 Release 2022/01/21
+            Version 2.1 Release 2022/06/08
```

Comment:
Maybe change this to Version 1.2, because in the targetNamespace is also still v1.

Reply:
Ok, modification done